controller/handler/trial.js

```diff
@@ -136,11 +154,12 @@ function trialView (request, reply) {
                         'Semicompliant',
                         'Noncompliant'
                     ]
-                })
+                }),
+                toDate: moment(fromDate).add(addSevenDays, 'days').format('YYYY-MM-DD')
```

Comment:
This will override any `toDate`, that the user has manually set.
This should preserve the original value.